ui/components/FilterableTable.tsx

```diff
@@ -153,12 +156,15 @@ export function initialFormState(cfg: FilterConfig, initialSelections?) {
 
 function applySelections(
   filters: FilterConfig,
-  selections: FilterSelections
+  initialSelections: FilterSelections
 ): FilterConfig {
-  if (!selections) {
-    return filters;
```

Comment:
this was the bigger bug about filtering too heavily to the properties of the initial data set.